demos/moderator_bot/tests/test_handlers.py

```diff
@@ -0,0 +1,16 @@
+async def test_listen_challenge(client):
+    resp = await client.post("/listen", json={"challenge": "challenge"})
+    result = await resp.text()
+    assert "challenge" in result
+
+
+async def test_list_message(client):
+    await client.post(
```

Comment:
can you explain this line please?

Reply:
No need to check response here as we just call slack api under the hood. But i will add request mocking to actually check that it was sent.